client/packages/coldchain/src/Monitoring/api/TemperatureNotification/hooks/document/useTemperatureNotifications.ts

```diff
@@ -22,8 +24,9 @@ export const useTemperatureNotifications = (queryParams: ListParams) => {
           warning(`${t('error.fetch-notifications')}: ${e.message}`)()
         ),
     {
-      cacheTime: 0,
-      refetchInterval: 1000 * 60 * POLLING_INTERVAL_IN_MINUTES,
+      cacheTime: POLLING_INTERVAL_IN_MILLISECONDS,
+      refetchInterval: POLLING_INTERVAL_IN_MILLISECONDS,
+      staleTime: STALE_TIME_IN_MILLISECONDS,
```

Comment:
Nit: is this more readable?:

```ts
      cacheTime: 3 * MINUTE,
      refetchInterval: 3 * MINUTE,
      staleTime: 1 * MINUTE,
```

With `const MINUTE = 60 * 1000;`

Reply:
it is more readable, yes. I prefer to have CONSTS at the top of the file as a rule, it means that if you want to change anything then you don't need to scroll down too far. having the numerical values lower in the code goes against that - although in this case, line 27 isn't that far to go!

I also prefer to have `MILLISECONDS` in the const name rather than just `MINUTE` because it isn't clear what units useQuery is expecting.

Reply:
In the js world everything is in `ms` on default, isn't it? But maybe just `_MS_` is enough here?

Reply:
I mean `STALE_TIME_IN_MILLISECONDS` -> `STALE_TIME_MS`

Reply:
mostly yes - but some packages helpfully have parameters in minutes or seconds. I've changed to `_MS` to make things neater 🙏